src/test/kotlin/uk/gov/justice/digital/hmpps/courthearingeventreceiver/controller/EventControllerIntTest.kt

```diff
@@ -41,6 +50,18 @@ class EventControllerIntTest : IntegrationTestBase() {
     courtCasesQueue?.sqsClient?.purgeQueue(PurgeQueueRequest.builder().queueUrl(courtCasesQueue!!.queueUrl).build())
     val str = File("src/test/resources/json/court-application-minimal.json").readText(Charsets.UTF_8)
     hearingEvent = objectMapper.readValue(str, HearingEvent::class.java)
+    amazonS3.createBucket(CreateBucketRequest.builder().bucket(bucketName).build())
```

Comment:
The other S3 bucket (used by the filter) is initialised in `setup-aws.sh` which means it is available to other tests and for local running if required.

Reply:
I see it's already in there - this is duplication of that